wrappers/python/c_files.cpp

```diff
@@ -139,6 +139,73 @@ void init_c_files(py::module &m) {
         .def_readwrite("tracker_confidence", &rs2_pose::tracker_confidence, "Pose confidence 0x0 - Failed, 0x1 - Low, 0x2 - Medium, 0x3 - High")
         .def_readwrite("mapper_confidence", &rs2_pose::mapper_confidence, "Pose map confidence 0x0 - Failed, 0x1 - Low, 0x2 - Medium, 0x3 - High");
+
+    py::class_<sc_float2> float2(m, "float2"); // No docstring in C++
+    float2.def(py::init<>())
+        .def_readwrite("x", &sc_float2::x, "x")
+        .def_readwrite("y", &sc_float2::y, "y");
+
+    py::class_<sc_float3> float3(m, "float3"); // No docstring in C++
+    float3.def(py::init<>())
+        .def_readwrite("x", &sc_float3::x, "x")
+        .def_readwrite("y", &sc_float3::y, "y")
+        .def_readwrite("z", &sc_float3::z, "z");
+
+    py::class_<sc_float3x3> float3x3(m, "float3x3"); // No docstring in C++
+    float3x3.def(py::init<>())
+        .def_readwrite("x", &sc_float3x3::x, "x")
+        .def_readwrite("y", &sc_float3x3::y, "y")
+        .def_readwrite("z", &sc_float3x3::z, "z");
+
+    py::class_<rs2_safety_extrinsics_table> safety_extrinsics_table(m, "safety_extrinsics_table"); // No docstring in C++
+    safety_extrinsics_table.def(py::init<>())
+        .def_readwrite("rotation", &rs2_safety_extrinsics_table::rotation, "Rotation Value")
+        .def_readwrite("translation", &rs2_safety_extrinsics_table::translation, "Translation Value");
+
+    py::class_<rs2_safety_preset_header> safety_preset_header(m, "safety_preset_header"); // No docstring in C++
+    safety_preset_header.def(py::init<>())
+        .def_readwrite("version", &rs2_safety_preset_header::version, "Version")
+        .def_readwrite("table_type", &rs2_safety_preset_header::table_type, "Table Type")
+        .def_readwrite("table_size", &rs2_safety_preset_header::table_size, "Table Size")
+        .def_readwrite("crc32", &rs2_safety_preset_header::crc32, "CRC32");
+
+    py::class_<rs2_safety_platform> safety_platform(m, "safety_platform"); // No docstring in C++
+    safety_platform.def(py::init<>())
+        .def_readwrite("transformation_link", &rs2_safety_platform::transformation_link, "Transformation Link")
+        .def_readwrite("robot_height", &rs2_safety_platform::robot_height, "Robot Height")
+        .def_readwrite("robot_mass", &rs2_safety_platform::robot_mass, "Robot Mass")
+        .def_property(BIND_RAW_ARRAY_PROPERTY(rs2_safety_platform, reserved, uint8_t, sizeof(rs2_safety_zone::reserved)), "Reserved");
+
+
+    py::class_<rs2_safety_zone> safety_zone(m, "safety_zone"); // No docstring in C++
+    safety_zone.def(py::init<>())
+        .def_readwrite("flags", &rs2_safety_zone::flags, "Flags")
+        .def_readwrite("zone_type", &rs2_safety_zone::zone_type, "Zone Type")
+        .def_property_readonly("zone_polygon", [](const rs2_safety_zone& self) { return reinterpret_cast<const std::array<sc_float2, sizeof(rs2_safety_zone::zone_polygon)>&> (self.zone_polygon);}, "zone polygon")
```

Comment:
Why is this readonly?

Reply:
putted this firstly to do some test, because could not convert the zone_polygon array natively to python list
working on this, will update it asap.